src/app/shared/common/language-selector/language-selector.component.html

```diff
@@ -0,0 +1,25 @@
+<button mat-icon-button [matMenuTriggerFor]="menu">
+  <mat-icon>
+    <img
+      *ngIf="selectedLanguage.value"
+      class="h-[1em]"
+      src="../../../../assets/images/flags/{{ selectedLanguage.value }}.svg"
+      alt="Selected Flag" />
```

Comment:
Shouldn't the alt text also be translated?

Reply:
Let's actually just remove the alt.